CRM/Upgrade/Incremental/php/FiveSixtySix.php

```diff
@@ -50,6 +50,36 @@ public function upgrade_5_66_alpha1($rev): void {
     $this->addTask('Update quicksearch options to v4 format', 'updateQuicksearchOptions');
   }
 
+  /**
+   * Upgrade step; adds tasks including 'runSql'.
+   *
+   * @param string $rev
+   *   The version number matching this function name
+   */
+  public function upgrade_5_66_beta1($rev): void {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    // Note these are localizable so need the last param
+    $this->addTask('civicrm_location_type.display_name default', 'alterColumn', 'civicrm_location_type', 'display_name', "varchar(64) NOT NULL DEFAULT '' COMMENT 'Location Type Display Name.'", TRUE);
```

Comment:
@demeritcowboy I wonder if given the PR by eileen only modified the translated columns definition i.e. not single lingual column definitions whether this should be all in an if statement and only done if we are in a multilingual setup?

Reply:
Hmm maybe I mixed something up. Ok let me revisit.